build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '0.1.167'
+version '0.1.168'
```

Comment:
With your changes the next version should be 169

Reply:
I was beaten to it. updated.